source/PlayerInfo.cpp

```diff
@@ -841,14 +841,30 @@ void PlayerInfo::AddShip(const shared_ptr<Ship> &ship)
 
 
 
-// Buy a ship of the given model, and give it the given name.
-void PlayerInfo::BuyShip(const Ship *model, const string &name)
+// Charge the player the cost of a ship and give it the specified name:
+void PlayerInfo::BuyShip(const Ship *model, const std::string &name)
+{
+	ReceiveShip(model, name, false);
+}
+
+
+
+// Receive a ship as a gift
```

Comment:
```suggestion
// Receive a ship of the give model with the given name as a gift.
```

Reply:
```suggestion
// Receive a ship of the given model with the given name as a gift.
```
the give model or given model?